app/templates/account/profile.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'layouts/base.html' %}
+
+{% block content %}
+    <div class="ui grid container">
+        <div class="ten wide computer sixteen wide mobile centered column">
```

Comment:
This should probably be `twelve wide` instead of `ten`. The account settings is twelve and we should try and keep it constant.